### ISSUE TITLE
[QUESTION]-Allow disabling SSL and hash checks when linking a plugin

### ISSUE THREAD

**Issue by rtoepfer** [forcedotcom/SFDX-Data-Move-Utility#326]
**Is your feature request related to a problem? Please describe.**

I get this error when trying to link the plugin to the CLI:

PS C:\Sandbox\SFDX-Data-Move-Utility> sfdx plugins:link                                                                                         
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.

(node:5912) Warning: Setting the NODE_TLS_REJECT_UNAUTHORIZED environment variable to '0' makes TLS connections and HTTPS requests insecure by disabling certificate verification.

(Use `node --trace-warnings ...` to show where the warning was created)

error An unexpected error occurred: "https://registry.yarnpkg.com/globby/-/globby-11.0.2.tgz: SSL Error: UNABLE_TO_GET_ISSUER_CERT_LOCALLY".

sfdx-cli: linking plugin sfdmu... !

    Error: yarn --non-interactive --mutex=file:C:\Sandbox\SFDX-Data-Move-Utility\yarn.lock

    --preferred-cache-folder=C:\Users\1219611\AppData\Local\sfdx\yarn --check-files exited with code 1

PS C:\Sandbox\SFDX-Data-Move-Utility>

**Describe the solution you'd like**
I'd like to have the ability to disable the "--check-files" option of yarn when linking a plugin to bypass the SSL errors due to ZScaler on corporate network.

**Comment by hknokh** [comment]
Hello @rtoepfer 
The plugins:link is the internal SFDX command and can't be altered or twicked. This feature can't be accomplished.
But found several articles discussing arround this kind of error message, i.e.:
https://stackoverflow.com/questions/45884752/npm-err-code-unable-to-get-issuer-cert-locally
Maybe you can find appropriate workarround.
Regards.

**Comment by rtoepfer** [comment]
I followed all the instructions in that article.  Also had to delete the yarn.lock file.  Then the installation from source worked.

**Comment by hknokh** [comment]
Hello @rtoepfer 
Thank you for the update.
Which one from the solution in that article worked for you ? What did you do ?

**Comment by rtoepfer** [comment]
@hknokh I set the regsitry to http (the non https), turned off node ssl check from command line, and set the node environment variable to turn SSL verify off.  Those were the three unique solutions on StackOverflow.

It seems sfdx/oclif hard codes the https yarn registry when doing the link if the yarn.lock file is present during an install from source.  I had to also delete the yarn.lock file after cloning the repo.  My corporate network uses Zscaler.
